scripts/fuzz_opt.py

```diff
@@ -718,7 +718,17 @@ class BinaryenInterpreter:
             name = 'binaryen interpreter'
 
             def run(self, wasm):
-                return run_bynterp(wasm, ['--fuzz-exec-before'])
+                output = run_bynterp(wasm, ['--fuzz-exec-before'])
+                if output != IGNORE:
+                    calls = output.count(FUZZ_EXEC_CALL_PREFIX)
+                    errors = output.count(TRAP_PREFIX) + output.count(HOST_LIMIT_PREFIX)
+                    if errors > calls / 2:
+                        # A significant amount of execution on this testcase
+                        # simply trapped, and was not very useful, so mark it
+                        # as ignored. Ideally the fuzzer testcases would be
+                        # improved to reduce this number.
+                        note_ignored_vm_run(f'(testcase mostly ignored: {calls} calls, {errors} errors)')
+                return output
```

Comment:
Should we also modify output to `IGNORE`?

Reply:
I think it's better not to, as then we'd still get some useful testing even if most functions trap. Though if it's 100% that's not much, obviously... but even 50% seems rare luckily.